README.md

```diff
@@ -56,7 +56,19 @@ list of prefixes:
 
     godepgraph -p github.com,launchpad.net bitbucket.org/foo/bar
 
-
+### As part of larger project (e.g. Docker):
+
+```
+mkdir -p go2/src/github.com/docker
```

Comment:
I just think the example is very verbose and quite unclear as to the intent and the "why" of all these commands